batch/sc-batch-fargate.yaml

```diff
@@ -0,0 +1,200 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: 'AWS Batch with Fargate'
```

Comment:
Is the reason to use fargate because it is a serverless option?

Reply:
yes exactly, using Fargate is almost like using lambda.  AWS will manage the compute instances that the tasks executes on so we don't have to manage it ourselves.